fastapi/encoders.py

```diff
@@ -95,7 +95,7 @@ def jsonable_encoder(
                 )
                 encoded_dict[encoded_key] = encoded_value
         return encoded_dict
-    if isinstance(obj, (list, set, frozenset, GeneratorType, tuple)):
+    if isinstance(obj, (list, set, frozenset, GeneratorType, tuple, Iterator)):
```

Comment:
Could we remove `list`, `set`, `frozenset`, `GeneratorType`, and `tuple`? I believe they are all iterables.

Reply:
Local testing verifies **list, set, and tuple** are all Iterators. But `frozenset` is not. I haven't been able to check on `GeneratorType`. Operating with a very restricted setup at the moment.